docs/interchange.rst

```diff
@@ -126,14 +138,20 @@ Column              Type                Description
 flags               uint32              Bitwise flags.
 time                double              Birth time of node
 population          int32               Birth population of node.
+individual          int32               The individual the node belongs to.
```

Comment:
perhaps these tables should have another column labeled "Optional?" for those that can be omitted.

Reply:
Too much work to do properly now; but feel free to open an issue to track it if you think it'd be good to do.